sdk/tests/fixtures/test_settings.toml

```diff
@@ -0,0 +1,521 @@
+# Description for the action
+#description = "Some edit action."
+# Arbitrary key/value pairs to store in the action.
+#template_parameters = { "arbitrary_key" = true }
+# A software agent has the same fields as a claim generator info.
+#software_agent = { name = "My Service" }
+
+# Multiple templates can be specified.
+#[[builder.actions.templates]]
+#action = "c2pa.cropped"
+
+# TODO: document rest of fields for actions
+# Actions to be added to every "Actions" assertion.
+#[[builder.actions.actions]]
+# The label of the action.
+#action = "c2pa.drawing"
+
+# Similarly, multiple actions can be defined.
+#382[[builder.actions.actions]]
+#action = "c2pa.color_adjustments"
+
+# Settings for configuring how c2pa.created actions are auto created.
+#
+# This is a convenience setting and it can be disabled if the information
+# is provided manually.
+[builder.actions.auto_created_action]
+# Whether to auto create the c2pa.created action.
+enabled = true
+# The source type field is required for the c2pa.created action.
+#
+# For more information, see `c2pa::assertions::actions::source_type`.
+source_type = "http://c2pa.org/digitalsourcetype/empty"
+
+# Settings for configuring how c2pa.opened actions are auto created.
+#
+# This is a convenience setting and it can be disabled if the information
+# is provided manually.
+[builder.actions.auto_opened_action]
+# Whether to auto create the c2pa.opened action.
+enabled = true
+# For more information, see `c2pa::assertions::actions::source_type`.
+#
+# Note this field is optional for the c2pa.opened action.
+source_type = "http://c2pa.org/digitalsourcetype/empty"
+
+# Settings for configuring how c2pa.placed actions are auto created.
+#
+# This is a convenience setting and it can be disabled if the information
+# is provided manually.
+[builder.actions.auto_placed_action]
+# Whether to auto create the c2pa.placed action.
+enabled = true
+# For more information, see `c2pa::assertions::actions::source_type`.
+#
+# Note this field is optional for the c2pa.placed action.
+source_type = "http://c2pa.org/digitalsourcetype/empty"
+
+# Settings for automatic thumbnail generation.
+[builder.thumbnail]
+# Whether to enable automatic thumbnail generation.
+enabled = true
+# Whether to ignore errors when generating a thumbnail and continue signing.
+ignore_errors = true
+# The size of the longest edge of the thumbnail.
+long_edge = 512
+# The output format of the thumbnail.
+#
+# If this field isn't specified, the thumbnail format will correspond to the
+# input format.
+#format = "png"
+# Whether or not to prefer a smaller sized media format for the thumbnail.
+#
+# The "format" option takes precedence over this field.
+#
+# For instance, if the source input type is a PNG, but it doesn't have an alpha channel,
+# the image will be converted to a JPEG of smaller size.
+prefer_smallest_format = true
+# The output quality of the thumbnail (low, medium, high).
+quality = "medium"
```

Comment:
Most of these things can be removed since they are defaults, besides `builder.actions.auto_created_action.source_type`/enabled. We can also set output quality to low and I think thumbnail.ignore_errors should be false too but we may run into issues there.

Reply:
I definitely ran into errors with thumbnail.ignore_errors set to false. Our default behaviour has been to just not generate thumbnails for unsupported formats. Tests for all those formats fail without this set to true.

Reply:
I can update this but I'd rather do it in another PR since this one has already grown so long. I'd like to have one standard test settings that is automatically preloaded whenever we are running tests,